tests/test_base.py

```diff
@@ -690,6 +699,23 @@ def test_base_index_access():
         h[0:10:20j, 0:5:10j, "hello", False, 5]
 
 
+def test_storage_proxy_int():
```

Comment:
You should have one test function per case. If you really want to bundle similar tests, use a class, but really, your naming scheme should do the bundling - if you want to run only `test_storage_proxy`, then you use `-k test_storage_proxy` and that will run this "group".